pkgs/data/themes/gruvbox-dark-gtk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-gtk";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1wf4ybnjdp2kbbvz7pmqdnzk94axaqx5ws18f34hrg4y267n0w4g";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/themes/
+    cp -r ${src} $out/share/themes/${pname}
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
```suggestion
  meta = with lib; {
```

Reply:
Is there any particular reason this method is preferred? They seem equivalent to me, but I might be missing something

Reply:
https://github.com/NixOS/nixpkgs/issues/108938